spec/requests/calculators_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Calculators', type: :request do
+  describe 'POST api/v1/calculators/PERMALINK/compute' do
+    it 'returns JSON' do
+      post '/api/v1/calculators/:id/compute'
```

Comment:
1. Instead of writing a string for the request URL, just use generated Rails route helper. See `rails routes | grep compute` for the exact name. 
2. Create a calculator record using factory and pass the `id` to route helper